pkg/scheduler/framework/types.go

```diff
@@ -75,80 +75,59 @@ const (
+	// AssignedPodDelete is the event when a pod is deleted that causes pods with matching affinity
+	// terms to be more schedulable.
+	AssignedPodDelete = ClusterEvent{Resource: Pod, ActionType: Delete, Label: "AssignedPodDelete"}
+	// NodeSpecUnschedulableChange is the event when unschedulable node spec is changed.
+	NodeSpecUnschedulableChange = ClusterEvent{Resource: Node, ActionType: UpdateNodeTaint, Label: "NodeSpecUnschedulableChange"}
+	// NodeAllocatableChange is the event when node allocatable is changed.
+	NodeAllocatableChange = ClusterEvent{Resource: Node, ActionType: UpdateNodeAllocatable, Label: "NodeAllocatableChange"}
+	// NodeLabelChange is the event when node label is changed.
+	NodeLabelChange = ClusterEvent{Resource: Node, ActionType: UpdateNodeLabel, Label: "NodeLabelChange"}
+	// NodeTaintChange is the event when node taint is changed.
+	NodeTaintChange = ClusterEvent{Resource: Node, ActionType: UpdateNodeTaint, Label: "NodeTaintChange"}
+	// NodeConditionChange is the event when node condition is changed.
+	NodeConditionChange = ClusterEvent{Resource: Node, ActionType: UpdateNodeCondition, Label: "NodeConditionChange"}
+	// PvAdd is the event when a persistent volume is added in the cluster.
+	PvAdd = ClusterEvent{Resource: PersistentVolume, ActionType: Add, Label: "PvAdd"}
+	// PvUpdate is the event when a persistent volume is updated in the cluster.
+	PvUpdate = ClusterEvent{Resource: PersistentVolume, ActionType: Update, Label: "PvUpdate"}
+	// PvcAdd is the event when a persistent volume claim is added in the cluster.
+	PvcAdd = ClusterEvent{Resource: PersistentVolumeClaim, ActionType: Add, Label: "PvcAdd"}
+	// PvcUpdate is the event when a persistent volume claim is updated in the cluster.
+	PvcUpdate = ClusterEvent{Resource: PersistentVolumeClaim, ActionType: Update, Label: "PvcUpdate"}
+	// StorageClassAdd is the event when a StorageClass is added in the cluster.
+	StorageClassAdd = ClusterEvent{Resource: StorageClass, ActionType: Add, Label: "StorageClassAdd"}
+	// CSINodeAdd is the event when a CSI node is added in the cluster.
+	CSINodeAdd = ClusterEvent{Resource: CSINode, ActionType: Add, Label: "CSINodeAdd"}
+	// CSINodeUpdate is the event when a CSI node is updated in the cluster.
+	CSINodeUpdate = ClusterEvent{Resource: CSINode, ActionType: Update, Label: "CSINodeUpdate"}
+	// ServiceAdd is the event when a service is added in the cluster.
+	ServiceAdd = ClusterEvent{Resource: Service, ActionType: Add, Label: "ServiceAdd"}
+	// ServiceUpdate is the event when a service is updated in the cluster.
+	ServiceUpdate = ClusterEvent{Resource: Service, ActionType: Update, Label: "ServiceUpdate"}
+	// ServiceDelete is the event when a service is deleted in the cluster.
+	ServiceDelete = ClusterEvent{Resource: Service, ActionType: Delete, Label: "ServiceDelete"}
+	// PodAdd is the event when a new pod is added to API server.
+	PodAdd = ClusterEvent{Label: "PodAdd"}
```

Comment:
Let's leave `PodAdd`, `ScheduleAttemptFailure` and `BackoffComplete ` as plain strings. Put them in a separate block.